engine/api/group/gorp_model.go

```diff
@@ -97,17 +97,17 @@ func (l LinksGroupProject) ToMapByProjectID() map[int64]LinksGroupProject {
 	return m
 }
 
-type Organization struct {
-	ID           int64  `db:"id"`
-	GroupID      int64  `db:"group_id"`
-	Organization string `db:"organization"`
+type GroupOrganization struct {
+	ID             string `db:"id"`
+	GroupID        int64  `db:"group_id"`
+	OrganizationID string `db:"organization_id"`
 	gorpmapper.SignedEntity
 }
 
-func (o Organization) Canonical() gorpmapper.CanonicalForms {
-	_ = []interface{}{o.ID, o.GroupID, o.Organization} // Checks that fields exists at compilation
+func (o GroupOrganization) Canonical() gorpmapper.CanonicalForms {
+	_ = []interface{}{o.ID, o.GroupID, o.OrganizationID} // Checks that fields exists at compilation
 	return []gorpmapper.CanonicalForm{
-		"{{print .ID}}{{print .GroupID}}{{.Organization}}",
+		"{{print .ID}}{{print .GroupID}}{{.OrganizationID}}",
```

Comment:
How can we migrate this ? No OrganizationOld like in user package ?

Reply:
I use setUserOrganization that ensureGroup organization automagically :)